demos/mkdocs/appyx-components/common/src/commonMain/kotlin/com/bumble/appyx/demos/common/AppyxWebSample.kt

```diff
@@ -54,6 +54,12 @@ fun <InteractionTarget : Any, ModelState: Any> AppyxWebSample(
     actions: Map<String, () -> Unit>,
     childSize: ChildSize = ChildSize.SMALL,
     modifier: Modifier = Modifier,
+    element: @Composable (ElementUiModel<InteractionTarget>) -> Unit = {
+        ModalUi(
+            elementUiModel = it,
+            isChildMaxSize = childSize == ChildSize.MAX
+        )
+    }
```

Comment:
It's not exactly the same as what you removed (that one had an alignment too), is this intended?